### ISSUE TITLE
inconsistent indent when argument is an array of callbacks?

### ISSUE THREAD

**Issue by isonmad** [eslint/eslint#7561]
**Tell us about your environment**

* **ESLint Version:** 3.9.1
* **Node Version:** 6.9.1
* **npm Version:** 3.10.8

**What parser (default, Babel-ESLint, etc.) are you using?**

**Please show your full configuration:**

```json
{
  "env": {
    "node": false,
    "browser": true
  },
  "rules": {
    "indent": ["warn", 2, { "SwitchCase": 1 }]
  },
  "globals": {
  }
}
```

**What did you do? Please include the actual source code causing the issue.**

```js
'use strict';
(() => {
  return Array.from([e => {
      throw e;
    },
    e => {
      throw e;
    }]);
})();
```

**What did you expect to happen?**
No warnings.

**What actually happened? Please include the actual, raw output from ESLint.**
  4:7  warning  Expected indentation of 4 spaces but found 6  indent
  5:5  warning  Expected indentation of 2 spaces but found 4  indent

This didn't seem to happen with earlier versions (last tested eslint 3.5.0). It can be avoided by putting a line break and a 4-space indent right after the `[`.

**Comment by not-an-aardvark** [comment]
Thanks for the report.

I think this is actually a duplicate of https://github.com/eslint/eslint/issues/6890. It started happening for arrays as well due to https://github.com/eslint/eslint/pull/7237.

**Comment by vitorbal** [comment]
Closing this as a duplicate of #6890. Please track that issue for updates on this bug. Thanks for the report, @isonmad!
